client/state/purchases/actions.js

```diff
@@ -13,15 +13,19 @@ import {
 	PURCHASE_REMOVE_FAILED,
 } from 'calypso/state/action-types';
 import { requestHappychatEligibility } from 'calypso/state/happychat/user/actions';
-
+import { requestAdminMenu } from '../admin-menu/actions';
+import { getSelectedSiteId } from '../ui/selectors';
```

Comment:
I cannot find now exactly where I read it, but I believe we have some guidelines in place that encourage us to use absolute paths when importing Redux actions/selectors:
```suggestion
import { requestAdminMenu } from 'calypso/state/admin-menu/actions';
import { getSelectedSiteId } from 'calypso/state/ui/selectors';
```